sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/GcpApiSurfaceTest.java

```diff
@@ -65,11 +65,13 @@ public void testGcpApiSurface() throws Exception {
             classesInPackage("com.google.bigtable.v2"),
             classesInPackage("com.google.cloud.bigquery.storage.v1beta1"),
             classesInPackage("com.google.cloud.bigtable.config"),
+            classesInPackage("com.google.common.collect"),
```

Comment:
You may want to be more specific here. This includes all the classes in Guava's collection, ImmutableList, ImmutableSet, etc. I believe Beam team has intentionally avoided the classes to be exposed as APIs.

Reply:
Ack, I will send another iteration to make it more specific.

Reply:
Done!